README.md

```diff
@@ -20,16 +20,32 @@ import PropTypes from 'prop-types'; // ES6
 var PropTypes = require('prop-types'); // ES5 with npm
 ```
 
-If you prefer a `<script>` tag, you can get it from `window.PropTypes` global:
+### CDN
 
+If you prefer to exclude `prop-types` from your application and use it 
+globally via `window.PropTypes`, the `prop-types` package provides 
+single-file distributions, which are hosted on the following CDNs:
+
+* [**unpkg**](https://unpkg.com/prop-types/)
+```html
+<!-- development version -->
+<script src="https://unpkg.com/prop-types@15.5/prop-types.js"></script>
+
+<!-- production version -->
+<script src="https://unpkg.com/prop-types@15.5/prop-types.js"></script>
+```
+
+* [**cdnjs**](https://cdnjs.com/libraries/prop-types)
 ```html
 <!-- development version -->
-<script src="https://unpkg.com/prop-types/prop-types.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/prop-types/15.5.10/prop-types.js"></script>
```

Comment:
Shouldn’t this be `15`?

Reply:
Nope ... and it's best practice to NOT use generic major versions.

Reply:
That wasn't the outcome of a collaborator's request [here](https://github.com/facebook/prop-types/pull/99#discussion_r133063401) - it should at least be `15.5`.

Reply:
It is specified for Unpkg which was the conversation and noted as also bad practice. CDNJS will always use exact versions -- better practice

Reply:
It wasn't specific to unpkg, but I didn't realize CDNJS doesn't support dropping minor/patch versions. I guess it's fine as-is, we don't release very often.

Reply:
Cool, thanks aweary. I have the same note/wording as on the Facebook page for consistency and I can send PRs every now and then to keep things up-to-date to help.